config/checkstyle.xml

```diff
@@ -9,313 +9,304 @@ page at http://checkstyle.sourceforge.net/config.html -->
-        <module name="NeedBraces">
-            <property name="severity" value="warning"/>
-            <!--<property name="tokens" value="LITERAL_IF, LITERAL_ELSE, LITERAL_FOR, LITERAL_WHILE, LITERAL_DO"/>-->
-            <property name="tokens" value="LITERAL_ELSE, LITERAL_FOR, LITERAL_WHILE, LITERAL_DO"/>
-        </module>
-
-        <module name="UpperEll">
-            <!-- Checks that long constants are defined with an upper ell.-->
-            <property name="severity" value="error"/>
-        </module>
-
-        <module name="FallThrough">
-            <!-- Warn about falling through to the next case statement.  Similar to
-            javac -Xlint:fallthrough, but the check is suppressed if a single-line comment
-            on the last non-blank line preceding the fallen-into case contains 'fall through' (or
-            some other variants which we don't publicized to promote consistency).
-            -->
-            <property name="reliefPattern"
-                      value="fall through|Fall through|fallthru|Fallthru|falls through|Falls through|fallthrough|Fallthrough|No break|NO break|no break|continue on"/>
-            <property name="severity" value="error"/>
-        </module>
-
-
-        <!--
-
-        MODIFIERS CHECKS
 
-        -->
+    <!--
+
+    MODIFIERS CHECKS
+
+    -->
+
+    <module name="ModifierOrder">
+      <!-- Warn if modifier order is inconsistent with JLS3 8.1.1, 8.3.1, and
+             8.4.3.  The prescribed order is:
+             public, protected, private, abstract, static, final, transient, volatile,
+             synchronized, native, strictfp
+          -->
+    </module>
 
-        <module name="ModifierOrder">
-            <!-- Warn if modifier order is inconsistent with JLS3 8.1.1, 8.3.1, and
-                 8.4.3.  The prescribed order is:
-                 public, protected, private, abstract, static, final, transient, volatile,
-                 synchronized, native, strictfp
-              -->
-        </module>
 
+    <!--
 
-        <!--
+    WHITESPACE CHECKS
 
-        WHITESPACE CHECKS
+    -->
 
+    <module name="WhitespaceAround">
+      <!-- Checks that various tokens are surrounded by whitespace.
+             This includes most binary operators and keywords followed
+             by regular or curly braces.
         -->
+      <property name="tokens" value="ASSIGN, BAND, BAND_ASSIGN, BOR,
+              BOR_ASSIGN, BSR, BSR_ASSIGN, BXOR, BXOR_ASSIGN, COLON, DIV, DIV_ASSIGN,
+              EQUAL, GE, GT, LAND, LE, LITERAL_CATCH, LITERAL_DO, LITERAL_ELSE,
+              LITERAL_FINALLY, LITERAL_FOR, LITERAL_IF, LITERAL_RETURN,
+              LITERAL_SYNCHRONIZED, LITERAL_TRY, LITERAL_WHILE, LOR, LT, MINUS,
+              MINUS_ASSIGN, MOD, MOD_ASSIGN, NOT_EQUAL, PLUS, PLUS_ASSIGN, QUESTION,
+              SL, SL_ASSIGN, SR_ASSIGN, STAR, STAR_ASSIGN"/>
+      <property name="severity" value="error"/>
+    </module>
+
+    <module name="WhitespaceAfter">
```

Comment:
There is no difference between this and the default, so the block can be simplified to `<module name="WhitespaceAfter"/>